app.js

```diff
@@ -176,7 +176,8 @@ app.all('/tests/*', (req, res) => {
       title: `${ident || 'All Tests'}`,
       layout: false,
       tests: foundTests,
-      hideResults: req.query.hideResults
+      hideResults: req.query.hideResults,
+      blacklist: (req.query.blacklist ? req.query.blacklist.split(',') : [])
```

Comment:
Can we call this `exclude` or `ignore`? It'll be in URLs and in the web standards space most instances of "blacklist" have been replaced by "blocklist", "ignorelist", or similar over the past year or so, to not link white/black to good/bad. In almost every case I think it's also been clearer after the change.

Reply:
Ahhhh, I wasn't aware that the terms blacklist/whitelist are phasing out, but I can definitely see why!  Following up now!